### DISCUSSION TITLE
thermistorLibrary issues

### DISCUSSION THREAD

**Discussion by alextrical** [aamott/petinator#15] [General]
Looking over the thermistorLibrary  it seems it isn't setup in a logical way.
It seems to reference 1 of 5 thermistors configured inside the library via its Configuration.h
The way you are trying to reference the thermistor doesn't look like its calling the table you want it to.

For example try to call the dummy table 998 that always returns 25C, instead it looks to be defaulting back to a B3950

**Comment by aamott** [comment]
That's very odd. At least it seems to work for the standard thermistor for now, but that should probably be replaced pretty soon. Thanks for noticing this!
